### PR TITLE
[BIT] fix paddle.unique

### PR DESCRIPTION
**Pull request by mzj104**
### 回测修复

回测时发现paddle.unique会有dtype diff的报错，修改了accuracy中对于paddle.unique的特殊处理，并将rule部分统一返回tuple类型，已全部通过，并移入accuracy_8.txt

### CODE REVIEW

File: tester/paddle_to_torch/rules.py [hunk 1]
```diff
@@ -5865,6 +5865,7 @@ def apply(self, paddle_api: str) -> ConvertResult:
             result[2] = result[2].to(dtype=dtype)
         else:
             result[1] = result[1].to(dtype=dtype)
+result = tuple(result)
```
Comment (cangtianhuang) [inline tester/paddle_to_torch/rules.py:5868]: 建议把result = list(result)放在if dtype is not None:中

File: tester/accuracy.py [hunk 2]
```diff
@@ -222,9 +222,10 @@ def test(self):
         if self.api_config.api_name == "paddle.incubate.nn.functional.fused_rms_norm": 
             paddle_output = paddle_output[0]
         if self.api_config.api_name == "paddle.unique": 
+            paddle_output = list(paddle_output)
             if "return_index=True" in self.api_config.config:
-                paddle_output = list(paddle_output)
                 del paddle_output[1]
+            paddle_output = tuple(paddle_output)
```
Comment (cangtianhuang) [inline tester/accuracy.py:228]: 建议改为：
```python
if self.api_config.api_name == "paddle.unique":
    if "return_index=True" in self.api_config.config:
        paddle_output = list(paddle_output)
        paddle_output.pop(1)
    paddle_output = tuple(paddle_output)
```

**Review by cangtianhuang — CHANGES_REQUESTED** [review]

**Review by cangtianhuang — APPROVED** [review]
LGTM
